### PR TITLE
feature: splash screen revamp

### PR DESCRIPTION
**Pull request by banklesss**
[YOMO-1286](https://emurgo.atlassian.net/browse/YOMO-1286)

[YOMO-1286]: https://emurgo.atlassian.net/browse/YOMO-1286?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/android/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <color name="bootsplash_background">#FFFFFF</color>
```
Comment (stackchain) [inline apps/wallet-mobile/android/app/src/main/res/values/colors.xml:2]: the splash screen will be white now 👀 ??

Comment (banklesss) [reply apps/wallet-mobile/android/app/src/main/res/values/colors.xml:2]: yeah !

File: apps/wallet-mobile/ios/nightly.plist [hunk 2]
```diff
@@ -88,11 +90,9 @@
 		<string>UIInterfaceOrientationLandscapeRight</string>
 		<string>UIInterfaceOrientationPortrait</string>
 	</array>
-	<key>UIViewControllerBasedStatusBarAppearance</key>
-	<false/>
-	<key>ITSAppUsesNonExemptEncryption</key>
-	<false/>
 	<key>UIUserInterfaceStyle</key>
 	<string>Light</string>
+	<key>UIViewControllerBasedStatusBarAppearance</key>
```
Comment (stackchain) [inline apps/wallet-mobile/ios/nightly.plist:95]: what does it do ?

Comment (banklesss) [reply apps/wallet-mobile/ios/nightly.plist:95]: nothing changed, just the order

**Review by stackchain — APPROVED** [review]
lg, 2 questions only.

File: apps/wallet-mobile/android/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/android/app/src/main/res/values/colors.xml:2]: https://www.figma.com/file/KeBVo6rWNYFLXwTphz9lHi/Mobile-App?node-id=11038%3A78867

**Pull request merged**